spec/common.js

```diff
@@ -7,7 +7,7 @@ process.env.ELASTICIO_API_USERNAME = 'user';
 process.env.ELASTICIO_API_KEY = 'apiKey';
 process.env.ELASTICIO_WORKSPACE_ID = 'workspaceId';
 
-const EXT_FILE_STORAGE = 'http://file.storage.server/file';
+const EXT_FILE_STORAGE = 'http://file.storage.server';
```

Comment:
Looks like logic change

Reply:
Was changed for proper work `nock` library (the library was updated)